### PR TITLE
Add widget to copy python file from client or server to job dir

### PR DESCRIPTION
**Pull request by jmancewicz**
Copies the python file to the job directory from the server of the client.

### CODE REVIEW

**Comment by lukeyeager** [comment]
Cool stuff! Can you post an example of how to use this?

**Comment by jmancewicz** [comment]
Sure. I'll try to make a simplified example.  Where do I post examples?

**Comment by lukeyeager** [comment]
> Where do I post examples?

Just to this thread is fine for now.

**Comment by jmancewicz** [comment]
Added a python layer test. @lukeyeager please take a look.

**Comment by lukeyeager** [comment]
Uh oh, the TravisCI build is failing with: 

    ERROR: Train Caffe Model: Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, Concat, ContrastiveLoss, Convolution, Data, Deconvolution, Dropout, DummyData, Eltwise, EuclideanLoss, Exp, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ReLU, Reshape, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, WindowData)

Looks like we need to add this line:

    sed -i 's/# WITH_PYTHON_LAYER/WITH_PYTHON_LAYER/g' Makefile.config

To [this file](https://github.com/NVIDIA/DIGITS/blob/v2.2.1/scripts/travis/install-caffe.sh#L28-L33) in order to get Caffe building with Python layers on Travis.

**Comment by lukeyeager** [comment]
@jmancewicz please remove the last commit. We'll address that in https://github.com/NVIDIA/DIGITS/pull/396. Since reverting https://github.com/NVIDIA/DIGITS/pull/394, you should be able to get the tests to pass here.
